datalab/datalab_session/utils/s3_utils.py

```diff
@@ -111,25 +121,22 @@ def get_archive_url(basename: str, archive: str = settings.ARCHIVE_API) -> dict:
   fits_url = results[0].get('url', 'No URL found')
   return fits_url
 
-def download_fits(file_path: str, basename: str, source: str = 'archive'):
-  basename = basename.replace('-large', '').replace('-small', '')
-  file_key = f"{source}_{basename}.fits.fz"
-  fits_path = os.path.join(settings.TEMP_FITS_DIR, file_key)
-  if not os.path.isfile(fits_path):
```

Comment:
I think you had mentioned the file_path param not being used @LTDakin - I just opted to use it here to cut out some of the duplicate code.